### PR TITLE
Ft home page

### PR DESCRIPTION
**Pull request by Elijahscriptdev**
Responsive design project using media queries to make it responsive on different screens

### CODE REVIEW

**Review by Forison — COMMENTED** [review]
# Pull request under review by Addo

**Review by Forison — CHANGES_REQUESTED** [review]
## Good job team🙂

I am impressed with your work but, here are a few changes you have to make to have your project approved.🙂

1. #### [REQUIRED] NAVBAR🙂
TNW logo must not be visible at large media screen🙂
> Look carefully at the behavior of the said (TNW)logo in the original app and replicate it as such.

2. #### [REQUIRED] TOP THREE ARTICLES🙂
The "Top 3 articles section" must contain TWN logo at the top left corner of the row left art.
>This is must be in the container with the class name ```.items1``` per your project.
> Look carefully at the behavior of the said logo in the original app and replicate it as such.

3. #### [REQUIRED] LATEST DEAL SECTION🙂
At small(max-width: 768px) screen latest deal must align 1 deal per row.🙂
> you can refer to the latest news section of your project 

4. Your ```.items2``` and ```.items3``` dimension are not making them look good.
> In the original site the dimensions of their heights combined is equal to the height of .```items1```
> study the behavior of these items in the original site and try as much as possible to replicate it.

## Make the required changes and submit them for another review.🙂
Happy Coding🙂🙂

**Comment by wathigo** [comment]
**Reviewing...**

File: styles.css
Comment (wathigo) [inline styles.css]: **[REQUIRED]** All the articles should have the same height. The first article should not contain logo in smaller screens. All articles should look similar in the smaller screens. Give them a dark background gradient to make the white text more visible

File: styles.css
Comment (wathigo) [inline styles.css]: **[REQUIRED]** The menu icon should appear on the right side of the navigation bar. We should have the search icon and menu icon on the left, TNW logo on the right for smaller screens

**Review by wathigo — CHANGES_REQUESTED** [review]
## Hi @Forison and @ampaire  :wave: 
### Great work so far!
Before we give this a :+1: , we need to fix some issues,
* All the articles should have the same height on smaller screens. The first article should not contain logo on smaller screens. All articles should look similar on the smaller screens. Give them a dark background gradient to make the white text more visible
* On smaller screens, both search icon and menu icon should be on the right side of the navigation bar, TNW logo on the  left.
* Please implement the changes pointed out below :point_down: 

**After implementing the changes, submit another request for review**

File: styles.css
Comment (wathigo) [inline styles.css]: **[REQUIRED]** Use a width of 100%

**Comment by ampaire** [comment]
This pull request fixes the feedback given by @wathigo . Correctio: @wathigo , I did the project with @Elijahscriptdev  and not @Forison . Thank you

**Review by stratospherique — CHANGES_REQUESTED** [review]
### Great work team :clap: . However, further fixes needs to be implemented in order for us to approve your project. check below :arrow_down: 

- Your page is having an horizontal scrolling effect on every screen width, you might have some overflowing content. :pray: fix it.

- The top article is now wider the other two below it. kindly fix it.

### That's it :relaxed: . When finished, kindly resubmit another code review request :pray:

**Review by uche-inyama — APPROVED** [review]
## Hi team, nice job,

### You have implemented the changes suggested by my colleagues. :slightly_smiling_face: 

But before you merge, take a look at these snap shots, use it as reference to adjust your sections from **latest news** section, there should be a space between the sections and your **container**, use bootstrap container as a reference.

[https://thenextweb.com/](https://thenextweb.com/)


Screenshot from 2019-09-24 18-59-29

[Your clone](https://rawcdn.githack.com/Elijahscriptdev/Responsive_design/17ccd801a72634f92a8b3fd1b7114cd0d9bbe2eb/index.html)

Screenshot from 2019-09-24 19-00-43

### Go ahead and merge to the master branch. :slightly_smiling_face:

**Pull request merged**
